entries/1/1password.com.json

```diff
@@ -14,6 +14,10 @@
     "custom-software": [
       "Duo"
     ],
+    "custom-hardware": [
+      "Yubico OTP",
+      "Duo D100 Token"
+    ],
```

Comment:
Is this documented anywhere?

Reply:
The Duo integration in 1Password requires the organization administrator to set up their own Duo organization. As such, any supported 2FA method that Duo offers _can technically_ be used as the user is the one to configure, set up and pay for the Duo service.